docs/guides/development/graphql/graphql-apollo-an-introduction/index.md

```diff
@@ -0,0 +1,230 @@
+
+## Apollo GraphQL vs REST
+
+An earlier standard for retrieval of Internet-hosted data was [representational state transfer](https://searchapparchitecture.techtarget.com/definition/REST-REpresentational-State-Transfer) (REST), defined by Roy Fielding’s 2000 [dissertation](https://www.ics.uci.edu/~fielding/pubs/dissertation/top.htm).  REST remains in wide use, and continues to have [advantages over GraphQL](https://blog.logrocket.com/why-you-shouldnt-use-graphql/#whyandwhentousegraphql) in straightforward applications and those based on web caching, among other situations.  Full comparisons of REST and GraphQL are well beyond the scope of this introduction, and several lengthy treatments are available, including one from Apollo itself.  Among the [benefits of GraphQL](https://www.mediacurrent.com/blog/5-reasons-why-you-should-consider-graphql-server/), you should be familiar with its potential to economize on network traffic and development effort, and to federate data from multiple and disparate sources.  REST implementations define a specific application programming interface (API), and requests from that API receive all its data.  The communications are determined by the server-side definitions.
+
+GraphQL dialogues, in contrast, receive only the data the client specifies.  This can be a small fraction of the networking load a REST architecture creates.  As a full-blown query language, GraphQL is more supple at expressing complex relations that potentially span multiple data sources.  REST-based applications express the same results [procedurally](https://www.toptal.com/software/declarative-programming), with all the potential for error that brings.
+
+The most important initial comparison of Apollo GraphQL and REST is to emphasize their similarity, as Apollo’s write-up mentioned in the previous paragraph properly does.  GraphQL and REST have much in common.
+
+## Apollo GraphQL Server
+
+Plenty of GraphQL [server implementations](https://blog.graphqleditor.com/graphql-servers) are available.  Apollo is a safe choice at the beginning of any project, if only for the completeness of its documentation and financial sustainability mentioned above.  Apollo is in active use by large projects including those under [recognizable names](https://stackshare.io/apollo) such as AirBnB, the New York Times, and CircleCI.
+
+### Server Installation Steps
+
+A minimal implementation based on Apollo’s server-side code is just as quick as the client example above.  Start with
+
+    npm install apollo-server-express express graphql
+
+Create `hello-server.js` with content
+
+{{< file "hello-server.js" >}}
+var {graphql, buildSchema} = require('graphql');
+
+var schema = buildSchema(`
```

Comment:
The backticks seem to confuse GitHub's rendering.  On my screen, the type switches to blue rather surprisingly.  It doesn't seem to hurt functionality, though.

Again, I'm ready to remove all the terminal semicolons.